### PR TITLE
build: definition who we are and styled mui

### PR DESCRIPTION
**Pull request by Heba-Adwan**
definition who we are and styled mui #28

### CODE REVIEW

File: client/src/components/Definition.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+//   StyledDefinitionSection,
+//   StyledTitle,
+//   StyledDescription,
+// } from './index';
+
+// interface DefinitionProps {
+//   title: string;
+//   description: string;
+// }
+
```
Comment (Saeed99Madi) [inline client/src/components/Definition.tsx:12]: remove the comment code

**Review by Saeed99Madi — CHANGES_REQUESTED** [review]
Great Job Hebe its looks you are professional, Just needs to remove the comment code

**Review by Emanalabsi — APPROVED** [review]

**Review by ahmedirheem — CHANGES_REQUESTED** [review]
For better file structure you must add the Component and its style in the same folder with the name of the component
for example:
-header|
             |-Header.tsx
             |-Header.styled.tsx

**Review by alaaahmade — APPROVED** [review]

File: client/src/components/Definition/DefinitionSection.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react';
+import {
```
Comment (alaaahmade) [inline client/src/components/Definition/DefinitionSection.tsx:2]: there is no index.ts file to import styled component from it

**Review by alaaahmade — CHANGES_REQUESTED** [review]

**Review by alaaahmade — APPROVED** [review]

**Review by ahmedirheem — APPROVED** [review]

**Pull request merged**